package.json

```diff
@@ -20,5 +20,9 @@
     "gulp-postcss": "^4.0.3",
     "gulp-sass": "^1.3.3",
     "gulp-sourcemaps": "^1.5.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "gulp"
```

Comment:
Oh, right. Makes sense. 
But I'm not sure a user needs to "start" the package or compile the Sass himself.